docs/variants/qemu_q35/development.md

```diff
@@ -0,0 +1,121 @@
+# Dasharo (UEFI) v0.1.0 for QEMU Q35
+
+
+1.  Please verify your qemu installation, and then invoke the qemu emulator with the built OVMF firmware image as given below:
```

Comment:
Would be good to give some references how to setup the QEMU installation.

Reply:
Will do that, documentation to QEMU installation will do?